book/generating-proofs/basics.md

```diff
@@ -13,3 +13,18 @@ To make this more concrete, let's walk through a simple example of generating a
 ```rust,noplayground
 {{#include ../../examples/fibonacci-io/script/src/main.rs}}
 ```
+
+## Build Script
+
+If you want your program crate to be built automatically whenever you build/run your script crate, you can add a `build.rs` file inside of `script/`:
+
+```rust,noplayground
+{{#include ../../examples/fibonacci-io/script/build.rs}}
+```
+
+Make sure to also add `sp1-helper` as a build dependency in `script/Cargo.toml`:
```

Comment:
should this be `sp1-core`?

Reply:
no, `sp1-helper` is its own crate so that build.rs doesn't need to compile lots of stuff.

Reply:
Oh, then can you make it `sp1-helper` below? it's listed as sp1-core :P

Reply:
o oops